package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "artillery-plugin-datadog",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Will bump the `minor` version due to `datadog-metrics` update and the change from `gauge` to `count`.